Description: remove redundant copy of test_cache_base_dir_exists
 Second copy of RbdTestCase.test_cache_base_dir_exists was accidentally
 introduced in https://review.openstack.org/82840.
Author: Dmitry Borodaenko <dborodaenko@mirantis.com>
Origin: upstream, https://github.com/angdraug/nova/commits/rbd-ephemeral-clone-stable-icehouse
Date: Thu, 3 Apr 2014 14:54:03 -0700
Change-Id: I16b2e472b2499f01379fdca9c860da0b95e291d8
Bug-Ubuntu: https://launchpad.net/bugs/1302211
Signed-off-by: Dmitry Borodaenko <dborodaenko@mirantis.com>
Last-Update: 2014-08-10

--- a/nova/tests/virt/libvirt/test_imagebackend.py
+++ b/nova/tests/virt/libvirt/test_imagebackend.py
@@ -804,20 +804,6 @@ class RbdTestCase(_ImageTestCase, test.NoDBTestCase):
 
         self.mox.VerifyAll()
 
-    def test_cache_base_dir_exists(self):
-        self.mox.StubOutWithMock(os.path, 'exists')
-        os.path.exists(self.TEMPLATE_DIR).AndReturn(True)
-        fn = self.mox.CreateMockAnything()
-        fn(target=self.TEMPLATE_PATH)
-        self.mox.StubOutWithMock(imagebackend.fileutils, 'ensure_tree')
-        self.mox.ReplayAll()
-
-        image = self.image_class(self.INSTANCE, self.NAME)
-        self.mock_create_image(image)
-        image.cache(fn, self.TEMPLATE)
-
-        self.mox.VerifyAll()
-
     def test_create_image(self):
         fn = self.prepare_mocks()
         fn(max_size=None, rbd=self.rbd, target=self.TEMPLATE_PATH)
-- 
2.1.1

